### ISSUE TITLE
Self-hosted: Events do not appear in funnel list

### ISSUE THREAD

**Issue by dubemarcantoine** [Swetrix/swetrix-fe#427]
Hi,
When trying to create a funnel, events do not appear in the funnel list. My installation is on Kubernetes. 

I tried with Docker compose provided and got the same error:
![Screenshot from 2024-06-08 10-12-17](https://github.com/Swetrix/swetrix-fe/assets/11211814/ba683c3d-1e60-4be8-bd97-15a154d25acd)

Error:
```json
{"message":"The provided type (ev) is incorrect","error":"Unprocessable Entity","statusCode":422}
```

docker-compose (essentially the one provided on your self-hosted docs):
```yaml
version: '3.9'
services:
  # This is the frontend, you can remove it if you want
  swetrix:
    image: swetrix/swetrix-fe:v3.0.0
    restart: always
    depends_on:
      - swetrix-api
    ports:
      - '80:3000'
    environment:
      - API_URL=http://localhost:8080/
      - SELFHOSTED=true

  # This is the analytics API
  swetrix-api:
    image: swetrix/swetrix-api:v3.0.0
    restart: always
    container_name: swetrix-api
    ports:
      - '8080:5005'
    environment:
      - JWT_ACCESS_TOKEN_SECRET=jwt-access-token-secret
      - JWT_REFRESH_TOKEN_SECRET=jwt-refresh-token-secret
      - REDIS_HOST=redis
      - CLICKHOUSE_HOST=http://clickhouse
      - API_ORIGINS=
      - EMAIL=test@test.com
      - PASSWORD=12345678
      - API_KEY=
      - CLOUDFLARE_PROXY_ENABLED=false
    links:
      - 'redis'
      - 'clickhouse'
    depends_on:
      - 'redis'
      - 'clickhouse'
  redis:
    image: redis:7.2-alpine
    restart: always
    environment:
      - REDIS_PORT=6379
      - REDIS_USER=default
      - REDIS_PASSWORD=
    ports:
      - 6379:6379
    volumes:
      - '/opt/redis-volume:/data'
  clickhouse:
    image: clickhouse/clickhouse-server:23.8-alpine
    container_name: clickhouse
    environment:
      - CLICKHOUSE_DATABASE=analytics
      - CLICKHOUSE_USER=default
      - CLICKHOUSE_PORT=8123
      - CLICKHOUSE_PASSWORD=
    ports:
      - 8123:8123
    restart: always
    volumes:
      - swetrix-events-data:/var/lib/clickhouse
    ulimits:
      nofile:
        soft: 262144
        hard: 262144
volumes:
  swetrix-events-data:
    driver: local
```

Thanks for your help!

**Comment by dubemarcantoine** [comment]
Oh and by the way I don't know if it could be worthwhile for you, but I could create a PR for the self-hosted to show how to deploy to K8s :smile: 

It works great apart from this issue!

**Comment by Blaumaus** [comment]
> Oh and by the way I don't know if it could be worthwhile for you, but I could create a PR for the self-hosted to show how to deploy to K8s 😄
> 
> It works great apart from this issue!

Hey! It would be amazing! Please do so when you have some time🙂

And regarding the bug, I'm taking a look at that, will let you know what might be wrong here

**Comment by Blaumaus** [comment]
This issue should be fixed in the [v3.0.1 API ](https://github.com/Swetrix/swetrix-api/releases/tag/v3.0.1)release

**Comment by dubemarcantoine** [comment]
Hi! Wow that was quick! 
Thank you so much for the quick work. I just pulled the new version in my K8s deployment and it works fine now 😁 

<img width="1428" alt="image" src="https://github.com/Swetrix/swetrix-fe/assets/11211814/5144a48c-da6e-4487-a29f-79f2a3deea47">


I will try to do this this week. I have to do some modifications to simplify the deployment because I configured Redis and Clickhouse separately in different namespaces, I'd rather for a self hosting doc have everything in the same namespace, the least configs possible. Should have some time this week 😄 I'll add a `k8s` folder in the selfhosting repo.
